arboretum/permissions/README.md

```diff
@@ -94,6 +94,53 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Checks
 
+### Organization Integrity (Repository Permissions)
+
+* Class: [OrgPermissionsCheck][org-permissions-check]
+* Purpose: Ensure that `direct` collaborators and forks do not exist in the organization repositories.
+* Behavior: Collaborators and forks are checked for every repository. A failure is generated when `direct` collaborators are found in a repository. A warning is generated for each fork found for a repository. 
+
+* Evidence depended upon:
+  * `direct` collaborators found in organization repositories.
+    * `raw/permissions/<gh|gl|bb>_direct_collaborators_<org_url_hash>.json`
+  * `outside` collaborators found in organization repositories.
+    * `raw/permissions/<gh|gl|bb>_outside_collaborators_<org_url_hash>.json`
+  * forks found in organization repositories.
+    * `raw/permissions/<gh|gl|bb>_forks_<org_url_hash>.json`
+  * Gathered by the permissions [GithubOrgPermissionFetcher][fetch-org-permissions]
```

Comment:
Remove this line to be consistent with the other check.

Reply:
Done